action.yaml

```diff
@@ -186,6 +196,13 @@ runs:
         set_env_var_if_provided "TRIVY_TF_VARS" "${{ inputs.tf-vars }}" ""
         set_env_var_if_provided "TRIVY_DOCKER_HOST" "${{ inputs.docker-host }}" ""
 
+    - name: Display Env variables
+      # Only when Actions Debug Logging enabled
+      if: ${{ runner.debug == '1' }}
+      shell: bash
+      run: |
+        cat trivy_envs.txt
+
```

Comment:
This information is needed now (while we are testing these changes).
In normal use, users don't need it. It can also be confusing (that we are using the same input for this debug and Trivy debug)

So let's just remove this logic.